sites/svelte.dev/content/blog/2023-07-01-whats-new-in-svelte-july-2023.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "What's new in Svelte: July 2023"
+description: "Svelte 4.0, New CHANGELOGs and a tour around the community"
+author: Dani Sandoval
+authorURL: https://dreamindani.com
+---
+
+Svelte 4 is out and folks have been building! There's a bunch of new showcases, libraries and tutorials to share. So let's get right into it...
+
+## What's new in Svelte
+The big news this month was the release of Svelte 4.0! You can read all about it in the [Announcing Svelte 4 post](https://svelte.dev/blog/svelte-4). From performance fixes and developer experience improvements to a brand new site, docs and tutorial... this new release sets the stage for Svelte 5 with minimal breaking changes.
+
+If you're already on Node.js 16, it's possible you won't see any breaking changes in your project. But be sure to read the [migration guide](https://svelte.dev/docs/v4-migration-guide) for all the details.
+
+For a full list of all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md) (note the new location in the `packages` directory)
```

Comment:
I see that as an "implementation detail" - is there a specific reason why you think it's important to note the changed location?
```suggestion
For a full list of all the changes to the Svelte compiler, including unreleased changes, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/packages/svelte/CHANGELOG.md).
```

Reply:
nope! It was relevant for me (since i read them all the time) but probably not important for most folks. good call on removing